rhcephcompose/compose.py

```diff
@@ -46,6 +46,8 @@ def __init__(self, conf):
         self.product_version = conf['product_version']
         # Extra files to put at the root of the compose
         self.extra_files = conf['extra_files']
+        # Whether sources composition should be skipped
+        self.no_sources = conf['no_sources']
```

Comment:
I get jumpy when I see too many cmdline arguments, so I was thinking we could just have Jenkins dynamically write this setting to the end of the Kobo conf file, or write a new "jenkins" kobo conf file (like we're currently doing for distill in Jenkins). What do you think?